corehq/apps/domain/templates/domain/manage_releases_by_app_profile.html

```diff
@@ -12,52 +12,63 @@
   {% initial_page_data 'appVersionOnlyShowReleased' True %}
   {% registerurl 'toggle_release_restriction_by_app_profile' domain '---'%}
   {% registerurl "paginate_releases" domain '---' %}
-
+<div class="container">
   <div class="row">
-    <div class="col-sm-12">
-      <div class="panel panel-modern-gray panel-form-only">
-        <div class="panel-body">
-          <form class="form-horizontal disable-on-submit" id="manage-app-releases" action=""
-                method='post'>
-            {% crispy manage_releases_by_app_profile_form %}
-          </form>
-          {% if app_releases_by_app_profile %}
-            <table id="managed-releases" class="table">
-              <thead>
-              <tr>
-                <th class="col-sm-2">{% trans 'Application' %}</th>
-                <th class="col-sm-2">{% trans 'Profile' %}</th>
-                <th class="col-sm-2">{% trans 'Version' %}</th>
-                <th class="col-sm-2">{% trans 'Status' %}</th>
-                <th class="col-sm-2">{% trans 'Action' %}</th>
-              </tr>
-              </thead>
-              <tbody data-bind="foreach: appReleasesByAppProfile, visible: appReleasesByAppProfile">
-              <tr data-bind="attr: {id: domId}, css: {'bg-success': active }">
-                <td data-bind="text: appName"></td>
-                <td data-bind="text: profileName"></td>
-                <td data-bind="text: version"></td>
-                <td data-bind="text: status"></td>
-                <td>
-                  <button type="button" class="btn default-btn"
-                          data-bind="click: toggleRestriction, css: buttonClass,
-                                            disable: ajaxInProgress">
-                    <span data-bind="text: actionText"></span>
-                    <i class="spinner fa-spin fa fa-refresh"
-                       data-bind="visible: ajaxInProgress"></i>
-                  </button>
-                  <br/>
-                  <span class="error-message label label-danger" data-bind="visible: error">
-                    {% trans 'Could not update!' %}
-                    <span data-bind="text: errorMessage"></span>
-                  </span>
-                </td>
-              </tr>
-              </tbody>
-            </table>
-          {% endif %}
-        </div>
+    <ul class="nav nav-tabs sticky-tabs" style="margin-bottom: 10px;">
```

Comment:
my guess is that you added a restriction sometime before and now you have deleted that version from the app? May be do an iteration on `LatestEnabledBuildProfiles.objects.all()` and check when `obj.build is None`?

Reply:
Thank you, got it working now, there was indeed a bad restriction 🙂 

I looked at this locally and think putting `<div class="spacer"></div>` below the ul is the best thing too do - I had thought there were other tab-related styles that would take care of that padding, but it doesn't seem like it.

While looking at this locally, I realized you can remove the wrapping `container` and `row` divs, and also it'd be nice to remove the legends, since now you have the tabs providing the same title & guidance.

Reply:
https://github.com/dimagi/commcare-hq/pull/25380/commits/0921ff721c26a883f393ae468ea673986eb90d3f